src/textual/validation.py

```diff
@@ -384,12 +384,15 @@ def __init__(
         minimum: int | None = None,
         maximum: int | None = None,
         failure_description: str | None = None,
+        strip_spaces: bool = True,
```

Comment:
Calling `strip` can potentially strip more than spaces. can we rename this to `strip_whitespace`?